data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/mapping/LogstashMappingModel.java

```diff
@@ -36,6 +36,10 @@
     @JsonSetter(nulls = Nulls.AS_EMPTY)
     private Map<String, Object> additionalAttributes = new HashMap<>();
 
+    @JsonProperty
+    @JsonSetter
```

Comment:
Is this line necessary?

Reply:
Good catch. It should actually be:

```
@JsonSetter(nulls = Nulls.AS_EMPTY)
```

This will create an empty `Map` when the YAML is:

```
defaultValues: null
```

Reply:
Thanks for catching this. Yes, it should create an empty Map when default values are null.